resource_handler.go

```diff
@@ -27,12 +28,21 @@ type ListRequestParams struct {
 // a resource based on the given attributes.
 type ResourceAttributes map[string]interface{}
 
+// Meta represents the metadata of a resource
+type Meta struct {
+	Created      *time.Time
+	LastModified *time.Time
+	Version      string
+}
```

Comment:
One last thing, could you add comments to the exported fields of `Meta`?

Reply:
Done!